pkgs/development/libraries/fcppt/default.nix

```
@@ -27,6 +33,6 @@ stdenv.mkDerivation rec {
     homepage = "https://fcppt.org";
     license = licenses.boost;
     maintainers = with maintainers; [ pmiddend ];
-    platforms = platforms.linux;
+    platforms = [ "x64_64-linux" "x86_64-windows" ];
```

it's a typo:
```suggestion
    platforms = [ "x86_64-linux" "x86_64-windows" ];
```

Alternatively you can do `broken = stdenv.targetPlatform.isAarch64; # revise when gcc is >= 10`